### ISSUE TITLE
Move the scout template editor to a global settings activity

### ISSUE THREAD

**Issue by vannaka** [SUPERCILEX/Robot-Scouter#108]
Create a new page in the app to put all of the settings. The scout templates should be put here so that they are in one centralized place. A scout should be able to be set as default. When the add scout button is pressed, it should automatically use the default scout. To choose which scout template to use just long press the add scout button and a list of the available scouts will pop up.  This would cut down on the number of clicks per action, it just minimizes annoyances. The setting could be navigated to by putting an option for it on the menu that is in the top right of the main activity that comes up when you tap on the three dots (not sure what those dots are called).

**Comment by vannaka** [comment]
Also, when creating a new template, it should start out empty. Currently, we are stuck with the predefined scout sheet.

**Comment by SUPERCILEX** [comment]
> To choose which scout template to use, just long press

@vannaka You just enlightened me there! 😄 I don't know why I couldn't figure it out, but I've been stuck on the UX of #82 for a while now. Thanks!

Yeah, I agreed that there should be a centralized place for settings. I didn't do that when I first started the app because there weren't any settings, but then I added a sign in feature, template editing, etc. and the settings grew. I'll definitely move them over for `v2.0`. (I didn't know that I had write access to other people's comments, but I edited your post to add a TODO list.)
